Doc/library/stdtypes.rst

```diff
@@ -4485,6 +4483,11 @@ The constructors for both classes work the same:
       Return ``True`` if the set has no elements in common with *other*.  Sets are
       disjoint if and only if their intersection is the empty set.
 
+   .. method:: set == other
+
+      Test whether every element of each set is contained in the other, that is,
+      ``set <= other and set >= other``.
```

Comment:
All objects implement `==` so it's redundant here IMO. In addition, we explain the differences with set == frozenset as well.